command/mounttune.go

```diff
@@ -0,0 +1,106 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/vault/vault"
+)
+
+// RemountCommand is a Command that remounts a mounted secret backend
+// to a new endpoint.
+type MountTuneCommand struct {
+	Meta
+}
+
+func (c *MountTuneCommand) Run(args []string) int {
+	var defaultLeaseTTL, maxLeaseTTL string
+	flags := c.Meta.FlagSet("mount-tune", FlagSetDefault)
+	flags.StringVar(&defaultLeaseTTL, "default-lease-ttl", "", "")
+	flags.StringVar(&maxLeaseTTL, "max-lease-ttl", "", "")
```

Comment:
I didn't implement a read on the CLI mount-tune command for two reasons: one, the info is shown in "mounts", and two, because due to the information that mount-tune is looking up you always get the current values (IOW, it's pulling the information from a SysView, not from the mount entries), not a "0" for the system default. In our previous conversation we decided to return 0 when the system default is being used. Otherwise you can get something back like a default of 180s and a max of 60s. Lease TTLs will be limited to the backend max, but it's showing the correct system default.